### PR TITLE
Initial commit of my prime factorization kata

### PR DESCRIPTION
**Pull request by fgs-dbudwin**
-Run from the terminal, enter a number to find it's factorization
-Ctrl+C to quit

### CODE REVIEW

File: src/Factorizer.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+namespace PrimeFactors.Src
+{
+    public class Factorizer
+    {
+        private const int FIRST_ODD_PRIME = 3;
+        private int currentPrimeNumber = FIRST_ODD_PRIME;
+
+        private List<int> primeFactors = new List<int>();
+
+        public void FindPrimeFactorsFor(int number)
```
Comment (FGS-CThomas) [inline src/Factorizer.cs]: This function is very long and contains a lot of the core logic of your factoring.  Generally speaking, long functions are hard to test and hard to understand.  Consider breaking this apart into a series of smaller private functions.

File: test/FactorizerTest.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public class FactorizerTest
+    {
+        [TestCase(2, new int[] { 2 })]
+        [TestCase(3, new int[] { 3 })]
+        [TestCase(22, new int[] { 2, 11 })]
+        [TestCase(74, new int[] { 2, 37 })]
+        [TestCase(90, new int[] { 2, 3, 3, 5 })]
+        [TestCase(9999, new int[] {3, 3, 11, 101 })]
+        [TestCase(360, new int[] { 2, 2, 2, 3, 3, 5 })]
+        public void TestFactoization(int numberToTest, int[] primeFactors)
```
Comment (FGS-CThomas) [inline test/FactorizerTest.cs]: Why these specific test cases?

Comment (fgs-dbudwin) [inline test/FactorizerTest.cs]: What do you mean?  Should I not test the factorization?

Comment (FGS-CThomas) [inline test/FactorizerTest.cs]: It's great to test the factorization but there are literally an infinite number of integers (ok, memory limits this but it's a very big number) you could pick from, so why those? Your tests are your documentation of your code; the test cases you pick represent something that I, as a reader, am supposed to understand.  What are these communicating?

**Pull request closed without merging**